### PR TITLE
updated package.json in just-animate, adding 1.0.3

### PR DESCRIPTION
**Pull request by notoriousb1t**
- updated name, website, and main file for just-animate
- including files from 1.0.3 since the file structure has changed from 1.0.2 to support typescript+webpack

### CODE REVIEW

File: ajax/libs/just-animate/package.json [hunk 1]
```diff
@@ -11,18 +11,19 @@
...
     "type": "git",
     "url": "https://github.com/just-animate/just-animate"
   },
   "npmName": "just-animate",
   "npmFileMap": [
     {
-      "basePath": "browser",
+      "basePath": "dist",
       "files": [
         "**/!(*.ts)"
```
Comment (pvnr0082t) [inline ajax/libs/just-animate/package.json]: @notoriousb1t I think we can replace `"**/!(*.ts)"` with `"**/!(index.js)"`here.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@notoriousb1t thank you!

**Pull request merged**